src/main/java/ToDo.java

```diff
@@ -0,0 +1,8 @@
+public class ToDo extends Task {
```

Comment:
Missing header comment for Todo class, AKA javadocs. (Under Comments in coding standard)